draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+  ecosystem.
+
+Governance and accountability are based on existing entities and legal
+mechanisms, and increasingly decentralized governance is strongly encouraged.
+
+
+## Motivation
+
+Starting at Zcash's second halving in November 2024, by default 100% of the
+block subsidies will be allocated to miners, and no further funds will be
+automatically allocated to any other entities. Consequently, no substantial new
+funding may be available to existing teams dedicated to furthering charitable,
+educational, or scientific purposes, such as research, development, and
+outreach.
+
+It is important to balance the incentives for securing the consensus protocol
+through mining with funding crucial charitable, educational, and scientific
+activities like research, development, and outreach. Additionally, there is a
+need to continue to promote decentralization and the growth of independent
+development teams.
+
+For these reasons, the Zcash Community wishes to establish a Zcash Hybrid
+Deferred Development Fund after the second halving in November 2024, and
+allocate 60% of the Dev Fund to Deferred Dev Fund “lockbox” and 40% to Zcash
+Community Grants.
+
+The primary motivations for this hybrid structure are:
+
+* **Avoiding immediate miner allocation**: Without the Hybrid Deferred Dev
+  Fund, the expiration of the current dev fund would result in the entire block
+  reward going to miners, requiring a future network upgrade to redirect funds
+  from miners to Dev Fund recipients, which could be controversial.
+
+* **Mitigating Regulatory Risks**: By minimizing direct funding of US-based
+  organizations, the lockbox helps to reduce potential regulatory scrutiny and
+  legal risks.
+
+* **Improving Funding Efficiency**: Allocating resources directly to projects
+  rather than organizations reduces inefficiencies and better aligns with
+  community priorities.
+
+* **Limited Runway**: ZCG does not have the financial runway that ECC/BP and ZF
+  have. As such, allocating ongoing funding to ZCG will help ensure the Zcash
+  ecosystem has an active grants program.
+
+* **Promoting Decentralization**: Allocating a portion of the Dev Fund to Zcash
+  Community Grants ensures small teams continue to receive funding to
+  contribute to Zcash. Allowing the Dev Fund to expire, or putting 100% into a
+  lockbox, would disproportionally impact grant recipients. This hybrid
+  approach promotes decentralization and the growth of independent development
+  teams.
+
+* **Moving Towards a Non-Direct Funding Model**: There is strong community
+  support for a non-direct Dev Fund funding model.  Allocating funds to a
+  Hybrid Deferred Dev Fund Lockbox gives the community time to research and
+  implement such a model while continuing to fund grant recipients.
+
+
+## Requirements
+
+**In-Protocol Lockbox Creation**: The ZIP must define the creation of an
+in-protocol lockbox where 20% of the Zcash block reward will be allocated
```

Comment:
This is incompatible with the abstract or the specification (which imply that 12% goes into the lockbox). For ease of reading, just remove the percentage from here:
```suggestion
in-protocol lockbox where a percentage of the Zcash block reward will be allocated
```